### DISCUSSION TITLE
Store Sesions MongoDB

### DISCUSSION THREAD

**Discussion by graficar** [devlikeapro/waha#283] [Q&A]
I am trying to implement with the new update, the storage of the sessions with mongodb that I have in Digital Ocean but apparently it gives an error in droplet that does not arise. How can I get more information about the connection and errors?

Thanks.

[![patron:PLUS](https://img.shields.io/badge/patron-PLUS-a0e6ba)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by allburov** [comment]
Hi! Could you check docker logs? Or it doesn't say anything there?
